hpx/util/unwrapped.hpp

```diff
@@ -495,20 +507,66 @@ namespace hpx { namespace util
                     util::detail::pack<typename std::decay<Ts>::type...>
                 >::type type;
         };
-    }
-
-    ///////////////////////////////////////////////////////////////////////////
+    } // end namespace detail
```

Comment:
All of the namespace detail should be wrapped in a pair of `/// \cond NOINTERNAL`/`/// \endcond`, please see the parallel algorithms for example. This ahides all the implementation detail from doxygen.